common/i18n/resources/en.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
     no: 'No',
     noItems: `$t(common.no) $t(common.item,{'count':2})`,
     of: 'of',
-    required: 'Required',
+    valueRequired: 'Required',
```

Comment:
why has the key changed? if the text says 'Required' I would call the key 'required' . note the keys in common are alphabetically ordered . also, is this still used?

Reply:
right